src/transformers/models/aria/modular_aria.py

```diff
@@ -904,7 +904,15 @@ def get_number_of_image_patches(self, height: int, width: int, images_kwargs=Non
         return num_patches
 
 
+class AriaImagesKwargs(ImagesKwargs, total=False):
+    split_image: Optional[bool]
+    max_image_size: Optional[int]
+    min_image_size: Optional[int]
```

Comment:
`Optional` means that the value can be set to `None`. Is this something we really want to allow? I would find it more natural to say "this field is not required but if it is set, then it's should be a `bool`/`int`.

e.g.

```suggestion
class AriaImagesKwargs(ImagesKwargs, total=False):
    split_image: bool
    max_image_size: int
    min_image_size: int
```

The `total=False` already tells that fields are not required

(same for all other kwargs defined)